crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -191,6 +211,76 @@ def clusters_healthy?
       def compute_resources_available?
         Api::Crowbar.compute_resources_available?
       end
+
+      # Check Errors
+      # all of the below errors return an array with the following schema:
+      # [
+      #   {
+      #     code: String,
+      #     data: ... whatever data type ...,
+      #     help: String # "this is how you might fix the error"
+      #   },
+      #   ...
+      # ]
+      def sanity_check_errors
+        return [] if network_checks.empty?
+
+        [
+          {
+            code: :network_checks,
```

Comment:
done. thanks for the hint. that made me remember, I still had to update the api documentation example...